package.json

```diff
@@ -253,7 +253,6 @@
     "react-refresh": "^0.10.0",
     "react-router-dom": "^6.21.2",
     "react-textarea-autosize": "^8.5.3",
-    "regenerator-runtime": "0.13.7",
```

Comment:
Duplicate

Reply:
I don't know what you mean by duplicate here, please revert it is not related to your PR

Reply:
It's needed to run Bun.js. Bun requires no duplicate package names.
This package is already declared in `devDeps`

Reply:
It's isn't in the dev deps though? `regenerator-runtime` appears in the package.json file one time only

Reply:
Not on master. It's in two places on master

Reply:
That's why this is a duplicate.

Reply:
It's in `dependiences` my bad. Not `devDeps`. It can only be called once in Bun.js or it throws an error.

Reply:
It just can't be in both `devDeps` and `deps`

Reply:
It must be a `dependience` for the CI to run.